src/clj/rems/api/applications.clj

```diff
@@ -176,15 +184,14 @@
     (GET "/handled" []
       :summary "Get all applications that the current user no more needs to act on."
       :roles #{:logged-in}
+      ;; XXX: checking this schema can be mighty slow (thousands of applications to check individually)
```

Comment:
there seems to be a newer version of compojure-api released: https://github.com/metosin/compojure-api/blob/master/CHANGELOG.md#200-alpha33-2024-04-30

though testing locally, e.g. resources api with performance data is still as slow as with older version.

Reply:
We should keep an eye on it an upgrade once there is a good "2.0" or so.